src/CoreWCF.ConfigurationManager/tests/WSHttpBindingTests.cs

```diff
@@ -63,8 +63,10 @@ public void WSHttpBinding_WithCustomSetting()
             }
         }
 
+#if !NETFRAMEWORK
```

Comment:
Why did you add this condition? Was it to stop the IDE from trying to run the test? The WCF client repo has a custom XUnit test runner with a lot of nice capabilities like forcing a crash dump with the test freezes. But I think that's overkill here. I found a nice approach in the [blog post here](https://josephwoodward.co.uk/2019/01/skipping-xunit-tests-based-on-runtime-conditions) where a custom attribute that derives from FactAttribute can be used. We could replace the Trait (and the command line arguments in the pipeline yml files) with a custom attribute which sets the Skip value.

Reply:
I've created an issue to open a follow up PR to implement the custom Fact attribute + pipeline modifications